test/Makefile

```diff
@@ -116,7 +116,18 @@ DEBUG_FLAGS=$(PIC_FLAG) -g
 
 export DMD_TEST_COVERAGE=
 
-runnable_tests=$(wildcard runnable/*.d) $(wildcard runnable/*.sh)
+# List the tests that take longest to run first, so that parallel make
+# will test them sooner, because they are large, have many test
+# permutations, or typically are the last tests to finish.
+runnable_tests_long=runnable/test42.d \
+		    runnable/xtest46.d \
+		    runnable/test34.d \
+		    runnable/test23.d \
+		    runnable/hospital.d \
+		    runnable/testsignals.d \
+		    runnable/interpret.d
```

Comment:
I'd probably add `runnable/sdtor.d` to this list as well.

Reply:
I don't recall seeing sdtor in the top 15 or so.

Reply:
Hmm, my memory must be off then.

Reply:
Indeed, sdtor is way down the list.  However, the reported run times look vastly different to what I saw last night as well.  We could get away with only test42.d and xtest46.d being in the list...

Number is seconds taken:
```
21 runnable/sdtor.d
22 runnable/test10942.d
22 runnable/testconst.d
27 runnable/test22.d
28 runnable/mixin1.d
31 runnable/argufilem.d
31 runnable/implicit.d
32 runnable/issue8671.d
32 runnable/test15568.d
33 runnable/inner.d
33 runnable/s2ir.d
33 runnable/test18545.d
33 runnable/test60.d
33 runnable/testsignals.d
34 runnable/builtin.d
34 runnable/lazy.d
35 runnable/foreach4.d
36 runnable/template4.d
36 runnable/template6.d
36 runnable/test15.d
37 runnable/arrayop.d
39 runnable/interpret.d
40 runnable/hospital.d
40 runnable/ifti.d
43 runnable/test28.d
45 runnable/test23.d
46 runnable/test34.d
143 runnable/test42.d
166 runnable/xtest46.d
```